### DISCUSSION TITLE
MQTT Config WebUI User/Password problem

### DISCUSSION THREAD

**Discussion by fmTechnix** [ClassicDIY/SkyeTracker#52] [Q&A]
Hey, thanks for this great project. 
I have configured MQTT. Unfortunately, I entered the wrong server IP. If I want to change this now in the WebUI-Congig, I need user and password. 

Isn't that ?
User:Skyetracker
Password:Skyetracker 

I can't get into the config with this data? 
Neither via IOS nor Android

**Comment by graham22** [comment]
the default should be 
User: admin 
Password: SkyeTracker

**Reply by fmTechnix** [reply]
Thank you
